src/main/java/org/apache/commons/configuration2/interpol/ConstantLookup.java

```diff
@@ -53,7 +53,7 @@
     private static final char FIELD_SEPRATOR = '.';
 
     /** An internally used cache for already retrieved values. */
-    private static Map<String, Object> constantCache = new HashMap<>();
+    private static final Map<String, Object> constantCache = new HashMap<>();
```

Comment:
@arturobernalg 
Again, static finals are in UPPER CASE by convention, so `constantCache` becomes `CONSTANT_CACHE`. PLEASE, review the whole PR before submission.

Reply:
Hi @garydgregory
I have double check and it seems that they are all changed already